bin/crew

```diff
@@ -520,7 +520,7 @@ def download
             cachefile = ''
           end
         else
-          puts 'Cannot find cached archive. 😔 Will download.'.lightred
+          puts "Cannot find cached archive at #{cachefile}. 😔 Will download.".lightred
```

Comment:
Not really an error but rather a warning?  Use yellow or orange instead of red?

Reply:
Done.